polkadot/runtime/parachains/src/dmp/tests.rs

```diff
@@ -170,6 +186,8 @@ fn dmq_pruning() {
 	let a = ParaId::from(1312);
 
 	new_test_ext(default_genesis_config()).execute_with(|| {
+		register_paras(&[a]);
```

Comment:
This PR is about checking the existence of parachains when DMP, and these tests are registering parachains; am I correct to infer that that this issue was detected here?

If so, can a couple of these tests be left as they were or replicated to keep testing for what happens when a parachain didn't exist?

Reply:
Done